.github/workflows/main.yaml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
```

Comment:
We should also run those on pull requests

Reply:
This is fixed in the latest commit.